packages/angular_devkit/build_angular/src/builders/web-test-runner/schema.json

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
Can you please keep up the schema, it looks like very few options are being used here.